ops/__init__.py

```diff
@@ -45,7 +45,6 @@
     '__version__',
     'main',
     'pebble',
-
```

Comment:
Does ruff force the lack of breaks here? If so, then that's ok, but if we can keep them I think we should, it helps show the logical sections.

Reply:
Although I think we should keep it since it increases readability, ruff unfortunately doesn't like it, and it seems there isn't a config about empty lines in imports, only a config to change the number of blank lines to place _after_ imports...

Reply:
Ok, let's let ruff do it's thing, then.